partiql-spi/src/main/kotlin/org/partiql/spi/internal/SqlTypeFamily.kt

```diff
@@ -99,5 +99,8 @@ internal class SqlTypeFamily private constructor(
                 PType.TIMESTAMPZ,
             )
         )
+
+        @JvmStatic
+        val DATE = SqlTypeFamily(PType.date(), setOf(PType.DATE))
```

Comment:
nit: for consistency with the other type families, let's include the call argument names

```suggestion
        val DATE = SqlTypeFamily(preferred = PType.date(), members = setOf(PType.DATE))
```

Reply:
ack